HW6/NyzkoshapkaOleksandr/task_02.py

```diff
@@ -0,0 +1,5 @@
+login = input("Input login: ")
```

Comment:
look at this implementation
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)